[JENKINS-62090] Specify SGs only on 1 part of request

res0nance opened the pull request:
Signed-off-by: Raihaan Shouhell <raihaan.shouhell@autodesk.com>

Specify setting on only 1 part of the request amends #447 

CC: @jzyeezy

MRamonLeon commented:
The error seems genuine:
```
[2020-05-08T03:36:54.131Z] [WARNING] Rule 5: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
[2020-05-08T03:36:54.131Z] Failed while enforcing RequireUpperBoundDeps. The error(s) are [
[2020-05-08T03:36:54.131Z] Require upper bound dependencies error for org.apache.commons:commons-compress:1.16.1 paths to dependency are:
[2020-05-08T03:36:54.131Z] +-org.jenkins-ci.plugins:ec2:1.51-rc1147.658b331984db
[2020-05-08T03:36:54.131Z]   +-org.testcontainers:testcontainers:1.8.3
[2020-05-08T03:36:54.131Z]     +-org.apache.commons:commons-compress:1.16.1
[2020-05-08T03:36:54.131Z] and
[2020-05-08T03:36:54.131Z] +-org.jenkins-ci.plugins:ec2:1.51-rc1147.658b331984db
[2020-05-08T03:36:54.131Z]   +-org.jenkins-ci.main:jenkins-core:2.222.3
[2020-05-08T03:36:54.131Z]     +-org.apache.commons:commons-compress:1.19
```
I introduced testcontainers to test connecting with certificates and so on

res0nance commented:
> The error seems genuine:
> 
> ```
> [2020-05-08T03:36:54.131Z] [WARNING] Rule 5: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
> [2020-05-08T03:36:54.131Z] Failed while enforcing RequireUpperBoundDeps. The error(s) are [
> [2020-05-08T03:36:54.131Z] Require upper bound dependencies error for org.apache.commons:commons-compress:1.16.1 paths to dependency are:
> [2020-05-08T03:36:54.131Z] +-org.jenkins-ci.plugins:ec2:1.51-rc1147.658b331984db
> [2020-05-08T03:36:54.131Z]   +-org.testcontainers:testcontainers:1.8.3
> [2020-05-08T03:36:54.131Z]     +-org.apache.commons:commons-compress:1.16.1
> [2020-05-08T03:36:54.131Z] and
> [2020-05-08T03:36:54.131Z] +-org.jenkins-ci.plugins:ec2:1.51-rc1147.658b331984db
> [2020-05-08T03:36:54.131Z]   +-org.jenkins-ci.main:jenkins-core:2.222.3
> [2020-05-08T03:36:54.131Z]     +-org.apache.commons:commons-compress:1.19
> ```
> 
> I introduced testcontainers to test connecting with certificates and so on

Yeah i just fixed it in master by excluding commons-compress from testcontainers
